speechmatics/models.py

```diff
@@ -283,6 +288,9 @@ class BatchTranscriptionConfig(_TranscriptionConfig):
     summarization_config: SummarizationConfig = None
     """Optional configuration for transcript summarization."""
 
+    sentiment_analysis_config: BatchSentimentAnalysisConfig = None
```

Comment:
The correct type for this is probably `Optional[BatchSentimentAnalysisConfig]`, but I see that's not done in the lines above either

Reply:
changed